amf-webapi/shared/src/main/scala/amf/plugins/document/webapi/parser/spec/declaration/OasTypeParser.scala

```diff
@@ -970,11 +977,17 @@ case class OasTypeParser(entryOrNode: YMapEntryLike,
 
       val property = producer(name)
         .add(Annotations(entry))
-        .set(PropertyShapeModel.MinCount, AmfScalar(if (required) 1 else 0), requiredAnnotations += ExplicitField())
+        .set(
+          PropertyShapeModel.MinCount,
+          AmfScalar(if (required) 1 else 0),
+          Annotations
+            .synthesized() += ExplicitField()) // this field is not explicit. In fact, explicit field annotation should disappear? Keeping annotation for oas to raml conversion tests.
 
       property.set(
         PropertyShapeModel.Path,
-        AmfScalar((Namespace.Data + entry.key.as[YScalar].text.urlComponentEncoded).iri(), Annotations(entry.key)))
+        AmfScalar((Namespace.Data + entry.key.as[YScalar].text.urlComponentEncoded).iri(), Annotations(entry.key)),
```

Comment:
Extract `entry.key.as[YScalar].text.urlComponentEncoded).iri()` to method

Reply:
The iri() method applies to all the composed Namespace:
(Namespace.Data + entry.key.as[YScalar].text.urlComponentEncoded)
No so much to reutilize extracting to a method. Already have common methods "+" , urlComponentEncoded and iri()